change/@azure-msal-browser-5533b82c-f8ab-46c4-93fb-040f7a99b3b7.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "[Native Auth] Fix the correlation id issue",
```

Comment:
"[Native Auth] Fix the correlation id issue" -> "[Native Auth] Fix the correlation id issue #8078"